packages/nep141-erc20/package.json

```diff
@@ -28,7 +28,7 @@
     "ethereumjs-util": "^7.0.10",
     "find-replacement-tx": "workspace:*",
     "lite-merkle-patricia-tree": "workspace:*",
-    "near-api-js": "^0.39.0",
+    "near-api-js": "https://github.com/near/near-api-js#fef7f683fca86980f3509987226fbc7f63b455cb",
```

Comment:
Open issue to track when https://github.com/near/near-api-js/pull/620 is merged

Reply:
https://github.com/aurora-is-near/rainbow-bridge-client/issues/46